src/cplscheme/CompositionalCouplingScheme.cpp

```diff
@@ -161,7 +161,7 @@ bool CompositionalCouplingScheme::hasDataBeenReceived() const
 double CompositionalCouplingScheme::getTime() const
 {
   PRECICE_TRACE();
-  auto schemes = allSchemes();
+  auto schemes = schemesToRun(); // analogous change to https://github.com/precice/precice/pull/1792, but if we use allSchemes() tests do not trigger. Why?
```

Comment:
@fsimonis I think this change would also make sense, but there is not really an observable effect. How to treat time with compositional coupling schemes is generally an important question as soon as we have moved to absolute timestamps.

Reply:
I just reconsidered this point: for `getTime()` using `allSchemes()` vs. `schemesToRun()` does not make a difference, if we take the minimum, because the schemes that are included in `allSchemes()` but not in `schemesToRun()` (i.e. the `convergedSchemes`) have already reached the end of the time window. Meaning: Their `getTime()` is always greater or equal to the time of the `schemesToRun()` and therefore does not matter for the minimum.

I'll remove this comment and we can go ahead.

Btw: Situation was different in #1792, because there we looked at the relative time, which is set to zero for converged schemes. Therefore, this influenced the minimum.